core/src/i18n/en.yaml

```
@@ -59,6 +45,17 @@ cron-jobs:
   title: Cron Jobs
 custom-resource-definitions:
   title: Custom Resource Definitions
```

It's spelled `CustomResourceDefinition`.

See the [Kubernetes documentation](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/). 

```suggestion
  title: CustomResourceDefinitions
```